seatunnel-plugin-discovery/src/main/java/org/apache/seatunnel/plugin/discovery/AbstractPluginDiscovery.java

```diff
@@ -114,6 +114,9 @@ public AbstractPluginDiscovery(
         this.pluginDir = pluginDir;
         this.pluginMappingConfig = pluginMappingConfig;
         this.addURLToClassLoaderConsumer = addURLToClassLoaderConsumer;
+        this.sourcePluginInstance = getAllSupportedPlugins(PluginType.SOURCE);
+        this.sinkPluginInstance = getAllSupportedPlugins(PluginType.SINK);
+        this.transformPluginInstance = getAllSupportedPlugins(PluginType.TRANSFORM);
```

Comment:
why we should cache this? It just store the connector jar prefix mapping. We can get the connector prefix when we used it.